samples/net/openthread/ncp/README.rst

```diff
@@ -1,16 +1,21 @@
 .. _ncp-sample:
 
-OpenThread NCP
+OpenThread Co-Processor
 ##############
```

Comment:
I have seen doc generator complain / create funny looking output if the `###...` is not the same length as the title text (not sure if this really issue atm). Anyway, please add more `#` chars here to be in the safe side.

Reply:
Changed.